db/changes.go

```diff
@@ -1281,6 +1288,13 @@ func createChangesEntry(ctx context.Context, docid string, db *DatabaseCollectio
 	row.Seq = SequenceID{Seq: populatedDoc.Sequence}
 	row.SetBranched((populatedDoc.Flags & channels.Branched) != 0)
 
+	if populatedDoc.HLV != nil {
+		cv := SourceAndVersion{}
+		cv.SourceID = populatedDoc.HLV.SourceID
+		cv.Version = populatedDoc.HLV.Version
+		row.CurrentVersion = &cv
+	}
```

Comment:
The method on `Document` I mentioned in previous comment would also be of use here.